<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCityGMLCut1</class>
 <widget class="QDialog" name="DialogCityGMLCut1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CityGMLCut</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">         
          <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
                <widget class="QLabel" name="label_GMLFiles">
                  <property name="text">
                    <string> CityGML files : </string>
                  </property>
                </widget>
              </item>
              <item>
                <layout class="QVBoxLayout" name="verticalLayout_IN">
                  <item>
                    <widget class="QLabel" name="label_IN">
                      <property name="text">
                        <string>IN</string>
                      </property>
                      <property name="alignment">
                        <set>Qt::AlignCenter</set>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QLineEdit" name="lineEdit_IN"/>
                  </item>
                </layout>
              </item>
              <item>
                <layout class="QVBoxLayout" name="verticalLayout_OUT">
                  <item>
                    <widget class="QLabel" name="label_OUT">
                      <property name="text">
                        <string>OUT</string>
                      </property>
                      <property name="alignment">
                        <set>Qt::AlignCenter</set>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QLineEdit" name="lineEdit_OUT"/>
                  </item>
                </layout>
              </item>
            </layout>
          </item>
          <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
                <widget class="QLabel" name="label_BoundingBox">
                  <property name="text">
                    <string> Bounding box : </string>
                  </property>
                </widget>
              </item>
              <item>
                <layout class="QVBoxLayout" name="verticalLayout_Xmin">
                  <item>
                    <widget class="QLabel" name="label_Xmin">
                      <property name="text">
                        <string>Xmin</string>
                      </property>
                      <property name="alignment">
                        <set>Qt::AlignCenter</set>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QLineEdit" name="lineEdit_Xmin"/>
                  </item>
                </layout>
              </item>
              <item>
                <layout class="QVBoxLayout" name="verticalLayout_Ymin">
                  <item>
                    <widget class="QLabel" name="label_Ymin">
                      <property name="text">
                        <string>Ymin</string>
                      </property>
                      <property name="alignment">
                        <set>Qt::AlignCenter</set>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QLineEdit" name="lineEdit_Ymin"/>
                  </item>
                </layout>
              </item>
              <item>
                <layout class="QVBoxLayout" name="verticalLayout_Xmax">
                  <item>
                    <widget class="QLabel" name="label_Xmax">
                      <property name="text">
                        <string>Xmax</string>
                      </property>
                      <property name="alignment">
                        <set>Qt::AlignCenter</set>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QLineEdit" name="lineEdit_Xmax"/>
                  </item>
                </layout>
              </item>
              <item>
                <layout class="QVBoxLayout" name="verticalLayout_Ymax">
                  <item>
                    <widget class="QLabel" name="label_Ymax">
                      <property name="text">
                        <string>Ymax</string>
                      </property>
                      <property name="alignment">
                        <set>Qt::AlignCenter</set>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QLineEdit" name="lineEdit_Ymax"/>
                  </item>
                </layout>
              </item>
            </layout>
          </item>
          <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
                <widget class="QLabel" name="label_Options">
                  <property name="text">
                    <string> Options : </string>
                  </property>
                </widget>
              </item>
              <item>
                <layout class="QVBoxLayout" name="verticalLayout_Verbose">
                  <item>
                    <widget class="QLabel" name="label_Verbose">
                      <property name="text">
                        <string>Verbose</string>
                      </property>
                      <property name="alignment">
                        <set>Qt::AlignVCenter|Qt::AlignJustify</set>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QCheckBox" name="checkBox_Verbose"/>
                  </item>
                </layout>
              </item>
            </layout>
          </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCityGMLCut1</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCityGMLCut1</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
